### PR TITLE
made it possible to register components on Angular Modules.

### PR DESCRIPTION
**Pull request by davidreher**
Since Angular 1.5 is already in RC state and this is a non conflicting addition, I would really love to see this added to the angular repository. The corresponding documentation can be found [here](https://docs.angularjs.org/api/ng/type/angular.Module)

### CODE REVIEW

**Comment by davidreher** [comment]
I know this is similar to #7222 but I have added more documented features ...

File: angularjs/angular.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for Angular JS 1.4+
+﻿// Type definitions for Angular JS 1.5.0.RC
 // Project: http://angularjs.org
-// Definitions by: Diego Vilar <http://github.com/diegovilar>
```
Comment (vvakame) [inline angularjs/angular.d.ts]: please keep original author's name. add your name below.

**Comment by vvakame** [comment]
please fix travis ci failed.

**Comment by davidreher** [comment]
I will create a new one ...

**Pull request closed without merging**